assets/src/scripts/pane.js

```diff
@@ -1,7 +1,7 @@
 window.hypothesisConfig = () => {
 	return {
-		openSidebar: pressbooksHypothesis.openSidebar,
-		showHighlights: pressbooksHypothesis.showHighlights,
+		openSidebar: ( pressbooksHypothesis.openSidebar === '1' ) ? true : false,
+		showHighlights: ( pressbooksHypothesis.showHighlights === '1' ) ? true : false,
```

Comment:
Heh, I had this same problem yesterday:

> [WP_Scripts::localize() encodes false as "" and true as "1"]

[You are doing the same.](https://github.com/pressbooks/pressbooks-book/blob/0a903b6427604ae62492f54e1f24c6f19f2b5954/inc/actions/namespace.php#L44)

For this reason I feel:

```
( pressbooksHypothesis.openSidebar ) ? true : false,
( pressbooksHypothesis.showHighlights) ? true : false,
```

Is a safer approach (in case one day `wp_localize_script` is more sensible)

Reply:
@connerbw Weirdly, I tried that and it doesn't work?